sdk/core/ts-http-runtime/test/authPolicy/apiKeyAuthenticationPolicy.spec.ts

```diff
@@ -84,6 +84,39 @@ describe("apiKeyAuthenticationPolicy", () => {
 
     vi.clearAllMocks();
   });
+
+  it.each([
```

Comment:
nit: I usually avoid using `it.each` because it adds cognitive overhead when you need to investigate a test failue, you have to parse the matrix to identify the failing case, and expectations often branch based on the index. In my experience, duplicating test code for readability is more valuable than saving a few lines. This is just a personal preference, so no need to change the test.